contexts/chat.tsx

```diff
@@ -186,12 +181,14 @@ const ChatContextProvider: React.FC<ChatContextProps> = ({
         if ((initialScript && initialScriptId) || threads.length === 0) {
           // if both threads and scriptId are set, then always create a new thread
           const newThread = await createThread(
-            script,
-            scriptDisplayName ?? defaultScriptName,
+            script !== '' ? script : tildy,
```

Comment:
can be `script ? script : tildy`? Are you specifially checking undefined option?

Reply:
I'll update this.

Reply:
Done.

Reply:
I know it's already merged, but a better solution is `script || tildy`. in JavaScript `''` is a falsy value